gitpod-io/gitpod
# [WS CLI] Make `env --scope=user` compatible with GitLab groups

## Description

Makes the "user scope" that the CLI lets you modify `*/**` instead of the previous `*/*`.

## Related Issue(s)

Fixes ENT-847

## How to test
0. In [your user settings](https://ft-gitlab-065c2d554b.preview.gitpod-dev.com/user/variables), create a new environment variable with the scopes `*/**`
1. [Open a workspace](https://ft-gitlab-065c2d554b.preview.gitpod-dev.com/new/#https://github.com/withastro/language-tools) 
2. Execute the following (in the workspace):
```
gp env foo=bar --scope user
```

This should update the contents of your variable's value to `bar`. You probably didn't name your variable `foo`, so change that part if you want to see it work.

Check the dashboard for the update or simply do `gp env`

For backwards compat, also try having a differently-named environment variable with the old `*/*` scope and try to update it with `gp env --user`. It should also work
